ibm/mas_devops/roles/suite_mustgather/tasks/main.yml

```diff
@@ -12,28 +12,56 @@
     that: base_output_dir is defined and base_output_dir != ""
     fail_msg: "base_output_dir property is required"
 
+# 2. Initialize parameters to determine what is collected
+# -----------------------------------------------------------------------------
+- name: "Collect optional parameters"
+  ansible.builtin.set_fact:
```

Comment:
Can we use just set_fact here rather than the fully qualified name

Reply:
changed